QUEST_LV_0200.tsv

```diff
@@ -6749,7 +6749,7 @@ QUEST_LV_0200_20150414_006752	We really owe you one.
 QUEST_LV_0200_20150414_006753	Thank you.{nl}I will find all ways to avoid opening the contaminated gap.
 QUEST_LV_0200_20150414_006754	How long can Aldona keep..
 QUEST_LV_0200_20150414_006755	If the Vakarine has received the Laima's warning in advance..{nl}The situation is better than now?
-QUEST_LV_0200_20150414_006756	Even I received the land from Gitise but it's all dirty.{nl}Especially, Ridimed. But, what can he do with the hand just got the pick?
+QUEST_LV_0200_20150414_006756	$I received land from Gitis, but it's all dirty.{nl}The Ridimeds are the problem. But, what can I do with just a hoe and pick?
```

Comment:
There are actually lots of Ridimeds :) He literally tells you this line while like 5 of them waddle around you and him